### DISCUSSION TITLE
How to create a gist FOLDER or MULTIPLE FILES in one gist?

### DISCUSSION THREAD

**Discussion by dbooth-boston** [community/community#29584] [Other Feature Feedback, Questions, & Ideas] [answered]
The [gist documentation](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists) does not say how to create a gist folder, or how to get multiple files into one gist.  (Grrrr.)   But obviously it is possible, because here is an example: https://gist.github.com/ericprud/8e53eef196ccdc2c43f40238fdd06691

Super frustrating that the documentation doesn't say how to do such a basic and important thing.   Anybody know how to do it please?

**Comment by mazkdevf** [comment]
Hello

![image](https://user-images.githubusercontent.com/79049205/184718673-8a9c7c8b-60f3-4378-88cd-a3dccf7973c0.png)

https://gist.github.com/

**Reply by mazkdevf** [reply]
if this is what you wanted and it is correct please mark this as answer so if someone else is looking also this they know the answer

**Comment by mazkdevf** [comment, marked as answer]
you just have to press that add file button to add new file as shown on the example what you linked

**Comment by airtower-luna** [comment]
Gists don't allow folders, you'll need a repository for that.

**Reply by dbooth-boston** [reply]
> Gists don't allow folders, you'll need a repository for that.

The above answer from @mazk5145 is what I was trying to find.  In essence, each gist is a folder that can have multiple files . . . if you can figure out how to add multiple file.  That's where I was stumped, and the documentation didn't help.

**Comment by earchibald-lv** [comment]
Easy peasy.

Where is says 'Embed', you can pull down to 'Clone via SSH' (or HTTPS).

clone the repository; adding other files in the top level will result in them being shown as in the example you gave.

**Reply by earchibald-lv** [reply]
Which is to say, gists *ARE* repositories, just dedicated-purpose ones with a front end.
